app/screens/home/StoreList.java

```diff
@@ -1,26 +1,68 @@
 package icebase.app.screens.home;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Scanner;
 import icebase.app.App;
+import icebase.app.api.API;
+import icebase.app.enums.SCREEN_ENUM;
 import icebase.app.MenuTitle;
+import icebase.app.Router;
 import icebase.app.screens.Screen;
+import icebase.app.types.Store;
+import icebase.icebase.Auth;
+import icebase.icebase.User;
 
 public class StoreList implements Screen {
     public void display() {
         Scanner sc = App.sc;
-
-        MenuTitle.displayMainTitle();
-        MenuTitle.displaySubTitle("[STORES]");
-        String[] storeList = { "Store 1", "Store 2" };
-        int range = storeList.length;
-        storeList[range - 1] = "Return";
         int choice;
+        Store chosenStore;
+        Auth currentUser = Auth.getAuth();
+
+        List<Store> stores = new ArrayList<>();
+        ArrayList<String> storeNames = new ArrayList<>();
+
+        // Gets names of each stores for options list
+        try {
+            stores = API.getStores();
+
+            for (Store store : stores) {
+                storeNames.add(store.getName());
+            }
+            storeNames.add("Return");
+
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+            return;
+        }
+
+        while (true) {
+            MenuTitle.displayMainTitle();
+            MenuTitle.displaySubTitle("STORES");
+            MenuTitle.displayOptions(storeNames);
```

Comment:
maybe maka add tayo ng print statement above the display options nga checking if may stores ba available?

smth like `No Stores Available :( pakyu`

we can do this by simply doing an if statement above this

```
// Equals 1 kay may return tayo nga option nga gin add manually
if (storeNames.size() == 1) {
    System.out.......
}
```

Reply:
I think sa new changes ni rian mae-ensure lagi na ang every user may store na macreate. tapos di rin makapunta ang user sa storelist unless di siya logged in or nakasign up

Reply:
ah ok got it, i guess ang item list lang siguro mo make sense ok
 will resolve